flow/storage/elasticsearch/elasticsearch.go

```diff
@@ -81,6 +81,54 @@ const flowMapping = `
 					"type": "date", "format": "epoch_millis"
 				}
 			}
+		},
+		{
```

Comment:
just declare these with simple regex like match by modify the "start"
like this : 
		{
			"start": {
				"match": "*Start",
				"mapping": {
					"type": "date", "format": "epoch_millis"
				}
			}
		},